tasks/config/karma.js

```diff
@@ -15,7 +15,7 @@ module.exports = function (grunt) {
       colors: true,
       logLevel: grunt.option('debug') || grunt.option('verbose') ? 'DEBUG' : 'INFO',
       autoWatch: false,
-      browsers: ['<%= karmaBrowser %>'],
+      browsers: ['Chrome'], // originally this line was: "browsers: ['<%= karmaBrowser %>'],"
```

Comment:
Was just experimenting with karma.  Wasn't sure how to use it until I discovered the JavaScript debugger keyword. I'll change this back when I'm finished.